### PR TITLE
Add embed comments route with only comments

### LINKED ISSUE #1624: Create page with only comments
Create a page `/embed/comments/@username/permlink` which include the comments component related to the username/permlink post. The page should not include any header or anything else, just this:
image

### PR DESCRIPTION
**Pull request by yamadapc**
This closes #1624

### CODE REVIEW

**Comment by bonustrack** [comment]
We will need transparent background aswell. We may want to make another Wrapper component to avoid too much conditional code. There is few things we dont need in Wrapper for the comment widget like getFollowing getNotifications getRebloggedList getTrendingTopics.

File: src/client/embed-post-comments/EmbedPostComments.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      state,
+      ownProps.match.params.author,
+      ownProps.match.params.permlink,
+    ),
+    loaded: getIsPostLoaded(state, ownProps.match.params.author, ownProps.match.params.permlink),
+    failed: getIsPostFailed(state, ownProps.match.params.author, ownProps.match.params.permlink),
+  }),
+  { getContent, getAccount },
+)
+export default class Post extends React.Component {
```
Comment (jm90m) [inline src/client/embed-post-comments/EmbedPostComments.js]: class name should match file name

File: src/client/embed-post-comments/EmbedPostComments.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    edited: false,
+    content: undefined,
+    fetching: false,
+    loaded: false,
+    failed: false,
+    getContent: () => {},
+    getAccount: () => {},
+  };
+
+  static fetchData(store, match) {
```
Comment (jm90m) [inline src/client/embed-post-comments/EmbedPostComments.js]: I don't think this is being used anywhere.

File: src/client/Wrapper.js [hunk 3]
```diff
@@ -188,3 +192,7 @@ export default class Wrapper extends React.PureComponent {
     );
   }
 }
+
+export function NoHeaderWrapper(props) {
```
Comment (jm90m) [inline src/client/Wrapper.js]: Wrapper contains lots of methods and imports that you won't probably use, better just to create a separate component

Comment (jm90m) [reply src/client/Wrapper.js]: oh nvm just saw @bonustrack comment

**Comment by bonustrack** [comment]
I'm closing this PR, we will do the comment widget within a new repo, these changes will not be used.

**Pull request closed without merging**